src/dotnet/OpenAPI.Validator/Model/Utilities/ValidationUtilities.cs

```diff
@@ -375,9 +375,11 @@ public static string GetCamelCasedSuggestion(string name)
             => Regex.Replace(
```

Comment:
It looks like we need to update the comments for this method, to include the updated examples.

Reply:
You mean the ones after each regex section?

Reply:
lines https://github.com/Azure/azure-openapi-validator/pull/131/files/d018e7f04bb49b7add7b373f35eb15ad1f722fd9#diff-ca0f26932fbcc64f5f8b35b071880575R369 and https://github.com/Azure/azure-openapi-validator/pull/131/files/d018e7f04bb49b7add7b373f35eb15ad1f722fd9#diff-ca0f26932fbcc64f5f8b35b071880575R370

Reply:
FIxed